backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/RunVmCommand.java

```diff
@@ -958,20 +956,11 @@ protected boolean getVdsToRunOn() {
         }
 
         addNumaPinning();
-        setDedicatedCpus();
+        setDedicatedCpus(getVdsManager());
```

Comment:
the changes in this command look alright but as discuss offline, let's make sure that we don't end up with leftovers within the topology in VdsManager in case RunVm fails after scheduling the host and before interacting with VDSM / when the call to VDSM fails